Explorer/Assets/Scripts/Global/Dynamic/DynamicSceneLoader.cs

```diff
@@ -166,12 +168,9 @@ async UniTask ChangeRealmAsync(string selectedRealm, CancellationToken ct)
 
                 await UniTask.SwitchToMainThread();
 
-                Vector3 characterPos = ParcelMathHelper.GetPositionByParcelPosition(settings.StartPosition);
-                characterPos.y = 1f;
-
-                staticContainer!.CharacterObject.Controller.transform.position = characterPos;
-
-                await realmController.SetRealmAsync(URLDomain.FromString(selectedRealm), ct);
+                var loadReport = new AsyncLoadProcessReport(new UniTaskCompletionSource(), new AsyncReactiveProperty<float>(0));
+                dynamicWorldContainer.MvcManager.ShowAsync(SceneLoadingScreenController.IssueCommand(new SceneLoadingScreenController.Params(loadReport))).Forget();
+                await realmController.SetRealmAsync(URLDomain.FromString(selectedRealm), settings.StartPosition, loadReport, ct);
             }
 
             ChangeRealmAsync(selectedRealm, CancellationToken.None).Forget();

```

Comment:
There was a mistake here: pass `destroyCancellationToken` (it's already embedded into `MonoBehavior`) instead of `None`